### PR TITLE
Merging pALPIDE-3

### PR DESCRIPTION
**Pull request by freidt**
Hi @mkofarag, hi @hyeonjoongkim,

please have a look at the changes of converter restoring the backward-compatibility.

Could you do some analysis test runs?

If you agree, I would make a pull request to the main repository.

Cheers,
Felix

### CODE REVIEW

File: producers/palpidefs/src/PALPIDEFSProducer.cxx [hunk 1]
```diff
@@ -427,15 +430,15 @@ void DeviceReader::Loop() {
       event_waiting = false;
 
     if (!event_waiting) {
-      // std::cout << "No event " << m_daq_board->GetNextEventId() << " " <<
-      // m_last_trigger_id << std::endl;
+//      std::cout << "No event " << m_daq_board->GetNextEventId() << " " <<
+//        m_last_trigger_id << std::endl;
```
Comment (mkofarag) [inline producers/palpidefs/src/PALPIDEFSProducer.cxx]: Remove?

File: producers/palpidefs/src/PALPIDEFSProducer.cxx [hunk 2]
```diff
@@ -478,13 +482,17 @@ void DeviceReader::Loop() {
 
       if (HeaderOK && TrailerOK) {
         if (m_debuglevel > 2) {
+//        if (1) {
```
Comment (mkofarag) [inline producers/palpidefs/src/PALPIDEFSProducer.cxx]: Remove?

File: producers/palpidefs/src/PALPIDEFSProducer.cxx [hunk 3]
```diff
@@ -498,31 +506,31 @@ void DeviceReader::Loop() {
         // std::cout << (uint64_t) m_daq_board << " " << header.EventId << " "
         // << header.TimeStamp << std::endl;
         SingleEvent *ev =
-            new SingleEvent(length - 28, header.EventId, header.TimeStamp);
+          new SingleEvent(length - 28, header.EventId, header.TimeStamp);
```
Comment (mkofarag) [inline producers/palpidefs/src/PALPIDEFSProducer.cxx:504]: Remove commented lines?

**Comment by freidt** [comment]
@hyeonjoongkim, does it work for you? If yes, I will go ahead with the pull request.
@mkofarag confirmed it works for pALPIDE-1 data.

**Comment by hyeonjoongkim** [comment]
@freidt sorry for late reply, yes it worked for me!

**Pull request merged**